redirects.yaml

```
@@ -1,12 +1,15 @@
-/doc?: https://canonical-mir.readthedocs-hosted.com/stable/
-/doc/index.html?: https://canonical-mir.readthedocs-hosted.com/stable/
-/doc/cppguide?: https://canonical-mir.readthedocs-hosted.com/stable/_static/cppguide/
-/doc/component_reports.html?: https://canonical-mir.readthedocs-hosted.com/stable/explanation/component_reports/
-/doc/dso_versioning_guide.html?: https://canonical-mir.readthedocs-hosted.com/stable/reference/dso_versioning_guide/
-/doc/getting_and_using_mir.html?: https://canonical-mir.readthedocs-hosted.com/stable/tutorial/
-/doc/getting_involved_in_mir.html?: https://canonical-mir.readthedocs-hosted.com/stable/how-to/getting_involved_in_mir/
-/doc/introducing_the_miral_api.html?: https://canonical-mir.readthedocs-hosted.com/stable/reference/introducing_the_miral_api/
-/doc/md_doc_kernel_requirements.html?: https://canonical-mir.readthedocs-hosted.com/stable/reference/kernel_requirements/
-/doc/deprecated.html?: https://canonical-mir.readthedocs-hosted.com/stable/api/page_deprecated/
-/doc/md__h_a_c_k_i_n_g.html?: https://github.com/canonical/mir/blob/main/HACKING.md
-/doc/md__r_e_a_d_m_e.html?: https://github.com/canonical/mir/blob/main/README.md
+# redirect for old /doc paths
+doc/?: https://canonical-mir.readthedocs-hosted.com/stable/
+doc/index.*/?: https://canonical-mir.readthedocs-hosted.com/stable/
+doc/cppguide?: https://canonical-mir.readthedocs-hosted.com/stable/_static/cppguide/
+doc/component_reports.*/?: https://canonical-mir.readthedocs-hosted.com/stable/explanation/component_reports/
+doc/dso_versioning_guide.*/?: https://canonical-mir.readthedocs-hosted.com/stable/reference/dso_versioning_guide/
+doc/getting_and_using_mir.*/?: https://canonical-mir.readthedocs-hosted.com/stable/tutorial/
+doc/getting_involved_in_mir.*/?: https://canonical-mir.readthedocs-hosted.com/stable/how-to/getting_involved_in_mir/
+doc/introducing_the_miral_api.*/?: https://canonical-mir.readthedocs-hosted.com/stable/reference/introducing_the_miral_api/
+doc/md_doc_kernel_requirements.*/?: https://canonical-mir.readthedocs-hosted.com/stable/reference/kernel_requirements/
+doc/deprecated.*/?: https://canonical-mir.readthedocs-hosted.com/stable/api/page_deprecated/
+doc/md__h_a_c_k_i_n_g.*/?: https://github.com/canonical/mir/blob/main/HACKING.md
+doc/md__r_e_a_d_m_e.*/?: https://github.com/canonical/mir/blob/main/README.md
+#catch all for /doc
+doc/(?P<page>.*)/*: https://canonical-mir.readthedocs-hosted.com/stable/
```

Why need to capture the `<page>`?

Can't it be just this?
```suggestion
doc/.*: https://canonical-mir.readthedocs-hosted.com/stable/
```

---

Yes, that will also work looks a little cleaner